srcpkgs/cinnamon-session/template

```diff
@@ -1,15 +1,16 @@
 # Template file for 'cinnamon-session'
 pkgname=cinnamon-session
 version=3.8.2
-revision=2
-short_desc="The Cinnamon session handler"
+revision=3
 build_style=meson
+configure_args="-D with-gconf=false"
```

Comment:
it is generally done with -Doption=value

Reply:
updated to use said option, and hopefully I have fixed the last of the lint errors :)